pgcli/pgcompleter.py

```diff
@@ -218,8 +220,11 @@ def _match(item):
 
                 completions.append((sort_key, item, meta))
 
+        if sort_ranked:
```

Comment:
I don't think so. Note that the following call to `sorted` is sorting the completions based on the output of the `_match` function, which in the case of fuzzy matching is a numerical score. Does that make sense?